### PR TITLE
#headingchange

### PR DESCRIPTION
**Pull request by Shreyar888**

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@Shreyar888  Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

File: content/de/_index.html
Comment (dipesh-rawat) [inline content/de/_index.html:1]: /easycla

**Review by dipesh-rawat — COMMENTED** [review]
/easycla

**Review by Arhell — COMMENTED** [review]
/easycla

File: content/de/_index.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 {{< blocks/section id="video" background-image="kub_video_banner_homepage" >}}
 <div class="light-text">
-<h2>Die Herausforderungen bei der Migration von über 150 Microservices auf Kubernetes</h2>
+<h2>Die Herausforderungen bei der Migration is in the über 150 Microservices auf Kubernetes</h2>
```
Comment (utkarsh-singh1) [inline content/de/_index.html:40]: Hi @Shreyar888, This document seems to be in German , are you sure these are the right changes you have made?

**Comment by divya-mohan0209** [comment]
Hello @Shreyar888 , please review our [contribution guidelines](https://kubernetes.io/docs/contribute/) and ensure you sign the CLA before committing a PR. 
/close

**Pull request closed without merging**